### PR TITLE
✏️ Adding AAAI 2019, arXiv preprints, and correcting links to resolve #17 and #18

### LINKED ISSUE #17: Wrong link for: [Rashwan2018b] Online Structure Learning for Feed-Forward and Recurrent Sum-Product Networks
The correct link is:

http://papers.nips.cc/paper/7926-online-structure-learning-for-feed-forward-and-recurrent-sum-product-networks.pdf

### PR DESCRIPTION
**Pull request by hayesall**
* Add Butz2019: "Deep Convolutional Sum-Product Networks"
* Add Molina2019: "SPFlow: An Easy and Extensible Library for
    Deep Probabilistic Learning using Sum-Product Networks"
* Add Wolfshaar2019: "Deep Convolutional Sum-Product Networks
    for Probabilistic Image Representations"
* Fix Rashwan2018b: Missing link to paper, added this in

This Pull Request resolves Issue #17 and Issue #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -403,10 +411,12 @@ Sorted by [year](#year) or [topics](#topics)
      `FSPN`
    - [[Poon2011](#poon2011)]
    [**Sum-Product Networks: A New Deep Architecture**](http://spn.cs.washington.edu/papers/spn.pdf) `SPN`
-
```
Comment (hayesall) [inline README.md:406]: I tried keeping the diff as small as possible. The tab character here seemed to be causing some issues in my editor, so I removed the line.

File: README.md [hunk 2]
```diff
@@ -53,6 +54,12 @@ Sorted by [year](#year) or [topics](#topics)
 
 ### Year
 
+#### 2019
+
+   - [[Butz2019](#butz2019)] [**Deep Convolutional Sum-Product Networks**](https://www.aaai.org/Papers/AAAI/2019/AAAI-ButzC.3622.pdf) *AAAI 2019* [`modeling`](#modeling)
+   - [[Molina2019](#molina2019)] [**SPFlow: An Easy and Extensible Library for Deep Probabilistic Learning using Sum-Product Networks**](https://arxiv.org/abs/1901.03704) *preprint* [`applications`](#applications)
```
Comment (hayesall) [inline README.md:62]: I listed Molina et al. SPFlow under `applications`, though it might fit under a couple topics.

File: README.md [hunk 3]
```diff
@@ -982,7 +993,21 @@ _Zhao, Han and Gordon, Geoff and Poupart, Pascal_<br/>
 NIPS 2017</p>
 * <p id="zheng2018">
 [Zheng2018]<br/>
-_Zheng, Kaiyu and Pronobis, Andrzej and Rao, Rajesh P.N.<br/>
+_Zheng, Kaiyu and Pronobis, Andrzej and Rao, Rajesh P.N._<br/>
```
Comment (hayesall) [inline README.md:1000]: There was a missing `_` character that would have caused strange italicization.

**Comment by hayesall** [comment]
Added another commit for `Shao2019`: Conditional Sum-Product Networks: Imposing Structure on Deep Probabilistic Architectures

File: README.md (hunk 2 above)
Comment (arranger1044) [reply README.md:62]: Better to link it to code as well

**Comment by arranger1044** [comment]
Thanks a lot Alexander

**Pull request merged**